### DISCUSSION TITLE
Unable to find data in Response exhaust report

### DISCUSSION THREAD

**Discussion by sravani-64** [Sunbird-Obsrv/Community#96] [Issues] [answered]
Hi team,

We have setup reports 4.10 version. We are able to submit progress exhaust and response exhaust jobs from portal UI and triggered the cron job manually for both reports. We are able to see that progress exhaust report is generated with data and downloaded from portal UI as well.
But for response exhaust, report file is generated but could not see any data in the report. To verify this, we have performed following analysis.

Verified if the job configurations is pointing to right Redis, Elastic search and Cassandra DBs and found that the configurations are correct.
Verified if we have relevant data in Redis and Cassandra as per https://lern.sunbird.org/learn/product-and-developer-guide/lms-service/reports/on-demand-exhaust/response-exhaust
We were able to find the data in **user_enrolments**
![image](https://github.com/Sunbird-Obsrv/Community/assets/133327173/d423647d-c4a8-4e8c-8ddf-5d0e3e8510b9)

**user_activity_agg**
![image](https://github.com/Sunbird-Obsrv/Community/assets/133327173/555447f3-757d-4930-9c5f-d8e2ab4e89bc)

but no data was found in **assessment_aggregator**
![image](https://github.com/Sunbird-Obsrv/Community/assets/133327173/119ee4ef-ef93-4d1a-aa1e-9736db093532)

Could someone please help us to understand what might be causing this issue? Thank you

**Comment by manjudr** [comment]
@sravani-64  - Are we running the assessment aggregator streaming flink job?

**Reply by sravani-64** [reply]
@manjudr yes assessment aggregator flink job is running

**Reply by manjudr** [reply, marked as answer]
@sravani-64  - Please check with the learn team once.
